gammapy/modeling/models/spectral.py

```diff
@@ -282,30 +284,30 @@ def integral(self, energy_min, energy_max, **kwargs):
         else:
             return integrate_spectrum(self, energy_min, energy_max, **kwargs)
 
-    def integral_error(self, energy_min, energy_max, epsilon=1e-4, **kwargs):
+    def integral_error(self, energy_min, energy_max, n_samples=10000, **kwargs):
```

Comment:
the arguments are changed... a depreciation warning scheme is needed

Reply:
technically n_samples is not a new name for epsilon, epsilon is removed and we have only `deprecated_renamed_argument` not `deprecated_remove_argument`.
How to deal with that @registerrier ?

Reply:
should we write  something explicitly ?
```
from warnings import warn
if epsilon!=1e-4:
        warn('epsilon will be deprecated in v2.0', GammapyDeprecationWarning, stacklevel=2)
```